std/internal/cstring.d

```diff
@@ -41,7 +41,7 @@ import std.traits;
 import std.range;
 
 version(unittest)
-@property inout(C)[] asArray(C)(inout C* cstr) pure nothrow @nogc
+@property inout(C)[] asArray(C)(inout C* cstr) pure nothrow @nogc @trusted
```

Comment:
This function can't be memory safe. It could be passed a pointer to non-null-terminated character.